hack/ci/e2e-k8s.sh

```diff
@@ -73,6 +73,9 @@ build() {
   fi
   # make sure we have e2e requirements
   make all WHAT="cmd/kubectl test/e2e/e2e.test ${GINKGO_SRC_DIR}"
+
+  # Ensure the built kubecl is used instead of system
+  export PATH="${PWD}/bin/kubectl:$PATH"
```

Comment:
I don't think that works? For one thing Kubernetes doesn't build to `bin/`

Reply:
You're right, I was looking at the wrong dir. I _think_ it'll need to be `export PATH="${PWD}/_output/bin/kubectl:$PATH" . This is based on https://github.com/kubernetes/kubernetes/blob/master/build/root/Makefile#L45-L46 and running e2e-k8s.sh locally.

Reply:
Still isn't the right fix. Looking more. /hold

Reply:
check in the kubernetes/hack folder, there are multiple scripts that looks for the `kubectl` binary there

Reply:
:sweat:  `PATH="${PWD}/_output/bin/kubectl:$PATH"` vs `PATH="${PWD}/_output/bin:$PATH"`.... Too much time spent on realizing that. Update incoming.

Reply:
I forget if we can rely on this always being the case but we can start here. When you build dockerized vs not the k8s build system emits to different subpaths under _output, but they might both be linked to this path? In the past in kubernetes upstream we've had to use a small utility to check multiple locations.

If this is working in CI we can start here though.